packages/web-app/cypress/e2e/browse-all-pages.cy.js

```diff
@@ -0,0 +1,86 @@
+describe('Simple pages loader to check all major GC pages load correctly', () => {
+  Cypress.Commands.add('checkPageLoaded', () => {
+    // Very simple check to know if the app crashed (blank page) or not
+    // Generally, if an error occurs, the h1 tag is not populated and a blank page is displayed.
+    cy.get('h1')
+      .should('exist')
+      .should('not.be.empty');
+  });
+
+  it('home page', () => {
+    cy.visit('/');
+    cy.checkPageLoaded();
+  });
+
+  it('advanced search page', () => {
+    cy.visit('/search');
+    cy.get('.MuiTabs-root');
+    // There is no <h1> tag on the advanced search page so just click on the search tabs
+    cy.get('.MuiTabs-flexContainer > :nth-child(2)').click();
+    cy.get('.MuiTabs-flexContainer > :nth-child(3)').click();
+    cy.get('.MuiTabs-flexContainer > :nth-child(4)').click();
+  });
+
+  it('map page', () => {
+    cy.visit('/map');
+    // There is no <h1> tag on the map page so check if the leaflet container is present instead
+    cy.get('.leaflet-container');
+  });
+
+  it('entrance page', () => {
+    cy.visit('/entrances/35120'); // Entrance with lot of data
+    cy.checkPageLoaded();
+
+    cy.visit('/entrances/6085'); // Entrance with almost no data
+    cy.checkPageLoaded();
+  });
+
+  it('cave page', () => {
+    cy.visit('/caves/75363'); // Cave with lot of entrances
+    cy.checkPageLoaded();
+
+    cy.visit('/caves/6085'); // Cave with almost no data
+    cy.checkPageLoaded();
+  });
+
+  it('massif page', () => {
+    cy.visit('/massifs/490');
+    cy.checkPageLoaded();
+  });
+
+  it('document page', () => {
+    cy.visit('/documents/22695'); // Collection
+    // Not perfect since the <h1> tag is used to display a "Loading data" message... To improve or change the page behaviour
```

Comment:
We should stop using this pattern on certain pages. Moreover, displaying "Loading XXX data" in a <h1> tag is not relevant semantically speaking.

Reply:
I agree !

Reply:
Fixed ! 🎉